deepcell_toolbox/deep_watershed.py

```diff
@@ -102,15 +102,16 @@ def deep_watershed(outputs,
 
 
 def deep_watershed_mibi(model_output,
-                        min_distance=10,
+                        radius=10,
```

Comment:
Try not to remove inputs to the function, since that will break old versions calling this. If you want to support `radius` as new input:
- set `min_distance` to `None` by default
- check to see if `min_distance` is passed, if so log a deprecated warning
- somehow decide which is the right value to use.